### PR TITLE
Get company goals

### PR DESCRIPTION
**Pull request by chuksdozie**
get company goals tab this resolves issues #349

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -26,6 +26,12 @@ jobs:
...
+            cd ..
+            yarn install
+            pm2 restart server.js
+=======
             cd .. 
             yarn install 
-            pm2 restart server.js
\ No newline at end of file
+            pm2 restart server.js
+>>>>>>> e0f09c91a6eae2595c80f7171b5bdbf24e444c68
```
Comment (Asylcreek) [inline .github/workflows/main.yml]: This file is still in the merging state, please complete your local merging.

If you are not sure which side is correct, just pull from the staging branch

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (chuksdozie) [reply .github/workflows/main.yml]: ok, this stuff is over whelming at times, the whole thing is always changing

Comment (chuksdozie) [reply .github/workflows/main.yml]: please I am pulling and its saying it is up to date...

Comment (chuksdozie) [reply .github/workflows/main.yml]: i have pulled this stuff over and over its saying its up to task and when i try to push its telling me to pull first

**Comment by chuksdozie** [comment]
@Asylcreek I have fixed thw conflict please check

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -26,6 +26,12 @@ jobs:
             cd client/
             yarn install
             yarn build
+
+            cd ..
+            yarn install
+            pm2 restart server.js
+
```
Comment (Asylcreek) [inline .github/workflows/main.yml:33]: Remove these lines, they are duplicates

**Review by Asylcreek — CHANGES_REQUESTED** [review]

**Comment by Asylcreek** [comment]
There still is a conflict in the Home.jsx file

**Comment by chuksdozie** [comment]
I dont even know what to do again. please what is the conflict?  @Asylcreek

**Comment by Asylcreek** [comment]
Just go to the team's repo on Github. Find client/src/pages/Home.jsx. Copy the files, paste it in your local client/src/pages/Home.jsx. Then make the PR again

**Comment by chuksdozie** [comment]
so I should close this one?

**Comment by Asylcreek** [comment]
I don't think you have to. Just make the changes and push again

File: .github/workflows/main.yml (hunk 2 above)
Comment (Asylcreek) [reply .github/workflows/main.yml:38]: This still needs to be worked on

**Review by Asylcreek — CHANGES_REQUESTED** [review]

**Comment by chuksdozie** [comment]
@Asylcreek  please check now

**Comment by chuksdozie** [comment]
@Asylcreek there are currently no conflicts

**Comment by abiona01** [comment]
He requested a change in file main.yml. You're yet to do so

**Comment by chuksdozie** [comment]
> He requested a change in file main.yml. You're yet to do so

Sorry what are you talking about, the main.yml file is exactly what is on the staging branch like it was copy and paste @abiona01

**Comment by chuksdozie** [comment]
@abiona01 please check o. There are no conflicts  currently..

File: .github/workflows/main.yml (hunk 2 above)
Comment (Asylcreek) [reply .github/workflows/main.yml:38]: Please, go to the main.yml on our team's github page, copy the contents there and paste it into your local main.yml file.

**Review by Asylcreek — CHANGES_REQUESTED** [review]

**Comment by chuksdozie** [comment]
please @abiona01  and @Asylcreek  I  want to draw your attention to the source of the conflict in the main.yml file, it is caused by the improper alignment on the part of the staging file..... where we have
`#    push`
`#        branches`

**Comment by chuksdozie** [comment]
all the while I thought I was doing rubbish but the fact is that for every new PR that you merge, it automatically raises conflicts in everyones else PR because ofcourse they are no longer in sync with the staging branch again.

**Comment by chuksdozie** [comment]
@abiona01 @Asylcreek  please check

**Review by abiona01 — APPROVED** [review]

**Comment by chuksdozie** [comment]
this PR resolves issue #349 
the test link is right [here](https://goals.zuri.chat)

**Pull request merged**